Swap.cpp

```diff
@@ -0,0 +1,55 @@
+#include<iostream>
+using namespace std;
+
+void SwapRef(int &a, int &b) 
+{
+   int temp;
+   temp = a; 
+   a = b;    
+   b = temp; 
+     
+   return;
+}
+
+void SwapPoint(int *a, int *b)
+{
+
+   int temp;
+   temp = *a;   
+   *a = *b;     
+   *b = temp;   
+  
+   return;
+}
+
+int Func (int A[],int n)
```

Comment:
Why is the return type int ? its not returning anything but printing the array using pointer concept. Rename the function to void print(int A[] , int n)